crates/client/src/config.rs

```diff
@@ -94,8 +204,323 @@ impl Config {
         }
         throw!(Error::BadWorkspace)
     }
+    pub fn merge_with_cli(&mut self, cli_vars: &str) {
+        // FIXME: we split by whitespace without respecting escaping or quotes - same approach as honggfuzz-rs so there is no point to fix it here before the upstream is fixed
+        let mut splitted_in_vector: Vec<String> =
+            cli_vars.split_whitespace().map(str::to_string).collect();
+
+        for x in &mut self.hfuzz_run_args.hfuzz_run_args {
+            // no need to re-format these as the long/short formats are predefined by us
+            let short_opt = &x.0;
+            let long_opt = &x.1;
+
+            let mut index = 0;
+            while let Some(arg) = splitted_in_vector.get(index) {
+                match arg.strip_prefix(short_opt) {
+                    Some(val) if short_opt.len() > 1 => {
+                        // -t timeout
+                        // -t<timeout> also supported (without the space)
+                        // -v <no next input because flag signals true>
+                        if x.2 == "true" || x.2 == "false" {
```

Comment:
If the value had an Option<String> type, than you could check only if x.val.is_none()...